pgjdbc/src/main/java/org/postgresql/PGConnection.java

```diff
@@ -315,4 +315,19 @@
    * @since 42.2.6
    */
   @Nullable String getParameterStatus(String parameterName);
+
+  /**
+   * Turn on/off adaptive fetch for connection. Existing statements and resultSets won't be affected
+   * by change here.
+   *
+   * @param adaptiveFetch desired state of adaptive fetch.
```

Comment:
this comment seems redundant

Reply:
Could you explain why the comment seems redundant to you? In my opinion, removing that comment would break the convention in `PGConnection` interface, because all other methods have explanation of their purpose and that one would be without.

Reply:
It doesn't really provide any useful information. I agree none of the others do, however I think it we can take the opportunity to at least fix this one